docs/cloud/what-are-the-default-limits-for-temporal-cloud.md

```diff
@@ -0,0 +1,42 @@
+---
+id: what-are-the-default-limits-for-temporal-cloud
+title: Default limits for Temporal Cloud
+sidebar_label: Cloud limits
+description: The default experience in Temporal Cloud has limits, some of which are configurable.
+---
+
+<!--- What are the default limits for Temporal Cloud? --->
+
+The default experience in Temporal Cloud has the following limits.
+Some of these limits are configurable.
+
+<!--- What are the default limits for accounts in Temporal Cloud? --->
+
+## Account default limits
+
+|                                      | Default                | Maximum (self-serve) | Maximum (system)                     |
+| ------------------------------------ | ---------------------- | -------------------- | ------------------------------------ |
+| Audit Log Retention Period           | 7 days                 |                      | 7 days                               |
```

Comment:
Because our Retention Period applies very specifically to Workflow Execution data.

And it is scoped to a Namespace.
It is a proper feature of the system - as it configurable.

That makes it specific enough to our platform that it is now a proper term for us.